components/infieldbutton/stories/template.js

```diff
@@ -15,11 +15,60 @@ export const Template = ({
 	iconName = "Add",
 	isDisabled,
 	isInvalid,
+	isStacked,
 	tabIndex = 0,
 	...globals
 }) => {
-
-	return html`
+	return isStacked ? html`
+    <button
+      class=${classMap({
+        [rootClass]: true,
+        [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+        [`${rootClass}--top`]: typeof position !== "undefined",
+        [`${rootClass}--quiet`]: isQuiet,
+        "is-invalid": isInvalid,
+        ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+      })}
+      ?disabled=${isDisabled}
+      aria-haspopup="listbox"
+      type="button"
+      tabIndex=${tabIndex}
+      aria-label="add"
+    >
+      <div class="${rootClass}-fill">
+        ${Icon({
+          ...globals,
+          size,
+          iconName: "ChevronUp75",
+          customClasses: [`${rootClass}-icon`],
+        })}
+      </div>
+    </button>
+    <button
+      class=${classMap({
+        [rootClass]: true,
+        [`${rootClass}--size${size?.toUpperCase()}`]: typeof size !== "undefined",
+        [`${rootClass}--bottom`]: typeof position !== "undefined",
+        [`${rootClass}--quiet`]: isQuiet,
+        "is-invalid": isInvalid,
+        ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+      })}
+      ?disabled=${isDisabled}
+      aria-haspopup="listbox"
+      type="button"
+      tabIndex=${tabIndex}
+      aria-label="add"
+    >
+      <div class="${rootClass}-fill">
+        ${Icon({
+          ...globals,
+          size,
+          iconName: "ChevronDown75",
+          customClasses: [`${rootClass}-icon`],
+        })}
+      </div>
+    </button>
+  ` : html`
```

Comment:
Technically, you can view the top and bottom button variants separately through the controls, but there wasn't a way to view them stacked on top of each other, and it's not very obvious that the top and bottom variatns can be stacked, which seemed like it would be a loss when the docs site is removed.